docs/source/introduction.rst

```diff
@@ -20,10 +20,25 @@ You can install scona directly from the GitHub repository::
 
 If you want to edit scona it's recommended that you pass the ``-e`` flag to ``pip`` to install the package editably.
 
+Getting Started
```

Comment:
I haven't checked this, but I'm happy with all the text if you say it looks good @wingedRuslan 😺